VirtoCommerce.Platform.Web/Swagger/OAuth2SecurityRequirementsFilter.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Http;
+using System.Web.Http.Description;
+using Swashbuckle.Swagger;
+using VirtoCommerce.Platform.Core.Web.Security;
+
+namespace VirtoCommerce.Platform.Web.Swagger
+{
+    public class OAuth2SecurityRequirementsFilter : IOperationFilter
```

Comment:
The platform already had operation filter that assigns OAuth2 authentication to all operations - [`AssignOAuth2SecurityOperationFilter`](https://github.com/VirtoCommerce/vc-platform/blob/features/1490-swagger-oauth2/VirtoCommerce.Platform.Web/Swagger/AssignOAuth2SecurityOperationFilter.cs). If you're adding a new filter for this, old filter should be removed.

Reply:
I used optimized version of my filter since the existing one doesn't work correctly.